### ISSUE TITLE
Legacy preferences still appear in Settings

### ISSUE THREAD

**Issue by nemoDreamer** [silvestreh/atom-material-ui#251]
I love all the work that's been done in "Settings", but I'm also still seeing duplicates for all the similarly names prefs: "Accent color" and "UI Accent Color", Multiple mutually excluding "Roboto" prefs, etc...

Could this be because one has to manually clean up one's own `config.cson`? Could this be done automatically?

<img width="350" alt="material_ui_settings" src="https://cloud.githubusercontent.com/assets/9303/13812469/b4e28b26-eb52-11e5-9f93-5c789e9945e9.png">

```cson
  "atom-material-ui":
    altCmdPalette: false
    colors:
      abaseColor:
        red: 217
        green: 188
        blue: 128
        alpha: 1
      genAccent: true
    compactTreeView: false
    fonts:
      useRoboto: true
    panels:
      altCmdPalette: false
    rippleAccentColor: false
    tabs:
      compactTabs: true
      rippleAccentColor: true
      showTabIcons: "Show on all tabs"
    treeView:
      compactList: true
    ui:
      accentColor: "Cyan"
      slimScrollbar: true
    useRoboto: false
    useRobotoInUI: false
```

**Comment by silvestreh** [comment]
Yep, there used to be a piece of code that would reset a user's settings when upgrading from 0.x to 1.x but unfortunately it kept resetting their settings after upgrading because, for some reason, localStorage wasn't available.

Here's what you can do… open a dev console and type:

```javascript
atom.config.set('atom-material-ui');
```

That should reset your settings and get rid of the old ones.

**Comment by silvestreh** [comment]
This is fixed by 2ed519514b3daf9c30a1900df73566361d8b6e4a :smile:
